algorithmic_efficiency/workloads/cifar/cifar_jax/input_pipeline.py

```diff
@@ -0,0 +1,329 @@
+"""CIFAR10 input pipeline.
+
+Forked from Flax example which can be found here:
+https://github.com/google/flax/blob/main/examples/imagenet/input_pipeline.py
+and adjusted to work for CIFAR10.
+"""
+
+from flax import jax_utils
+import jax
+import tensorflow as tf
+
+IMAGE_SIZE = 32
+RESIZE_SIZE = 40
```

Comment:
is it standard to upscale the cifar images? I always used the original 32px ones.

Reply:
What I've mainly seen is upscaling by padding 4 on each side, then random crops, e.g. [pytorch example](https://pytorch-tutorial.readthedocs.io/en/latest/tutorial/chapter03_intermediate/3_2_2_cnn_resnet_cifar10/). 
However, it turns out _random_crop() here already does proper resizing of the crops, so I just removed the unnecessary RESIZE_SIZE and _center_crop().